src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -162,7 +169,23 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
       primaryLabel: {
         type: LabelSignalEnumType,
         description: "Primary label signal available to collector",
-        resolve: (artwork, {}, ctx) => getPrimaryLabel(artwork, ctx),
+        args: {
+          ignore: {
+            type: GraphQLList(LabelSignalEnumType),
+            description: "Signals to ignore",
+          },
+        },
+        resolve: (artwork, args, ctx) => {
+          const { ignore } = args
+          if (ignore?.length > 0) {
+            if (ignore.length > AVAILABLE_LABEL_COUNT) {
```

Comment:
There's no need for the double if's with a `?`

Reply:
Ah youre right. relic of an earlier commit.